internal/scaffold/ansible/watches_test.go

```diff
@@ -0,0 +1,109 @@
+package ansible
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/operator-framework/operator-sdk/internal/scaffold"
+	"github.com/operator-framework/operator-sdk/internal/util/fileutil"
+	"gotest.tools/assert"
+)
+
+func TestUpdateAnsibleWatchForResource(t *testing.T) {
+	mockData := "---" + "\n" +
+		"- group: mykind.example.com" + "\n" +
+		"  version: v1alpha1" + "\n" +
+		"  kind: Mykind" + "\n" +
+		"  role: mykind"
```

Comment:
nit: typically I use a raw multiline string. 
https://github.com/operator-framework/operator-sdk/blob/master/internal/scaffold/go_mod.go#L39-L52

So for this I would've done:
```golang
mockData = `---
- group: mykind.example.com
  version: v1alpha1
  kind: MyKind
  role: mykind`
```
Let's do this in a follow on.